src/Integration/Resources/Strings.resx

```diff
@@ -547,4 +547,42 @@ This is the general format to use when writing errors to output window or when t
   <data name="SolutionIsClosed" xml:space="preserve">
     <value>The operation requires an open solution.</value>
   </data>
+  <data name="SonarLintInfoBarUpdateCommandIsBusyRetry" xml:space="preserve">
+    <value>SonarLint: Could not update the binding since the command is not available at the moment. Please try again and make sure that not building or debugging.</value>
```

Comment:
> SonarLint: Could not update the binding because the command is not currently available. Please try again and ensure that the solution is not building or currently being debugged.